From: Jeff Mahoney <jeffm@suse.com>
Subject: [PATCH] gameport: unexport gameport_cooked_read and gameport_set_name

 gameport_cooked_read and gameport_set_name are both static inlines. Exporting
 them causes a section conflict.

Signed-off-by: Jeff Mahoney <jeffm@suse.com>

---

 drivers/input/gameport/gameport.c |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/input/gameport/gameport.c	2007-10-31 17:43:33.000000000 -0400
+++ b/drivers/input/gameport/gameport.c	2007-10-31 17:43:47.000000000 -0400
@@ -38,8 +38,6 @@ EXPORT_SYMBOL(gameport_unregister_driver
 EXPORT_SYMBOL(gameport_open);
 EXPORT_SYMBOL(gameport_close);
 EXPORT_SYMBOL(gameport_rescan);
-EXPORT_SYMBOL(gameport_cooked_read);
-EXPORT_SYMBOL(gameport_set_name);
 EXPORT_SYMBOL(gameport_set_phys);
 EXPORT_SYMBOL(gameport_start_polling);
 EXPORT_SYMBOL(gameport_stop_polling);
